pkgs/development/python-modules/hydra-check/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, docopt
+, requests
+, beautifulsoup4
+, black
+, mypy
+, flake8
+}:
+buildPythonPackage rec {
+  pname = "hydra-check";
+  version = "1.1.1";
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitHub {
+    owner = "nix-community";
+    repo = "hydra-check";
+    rev = version;
+    sha256 = "1dmsscsib8ckp496gsfqxmq8d35zs71n99xmziq9iprvy7n5clq2";
+
+  };
+
+  propagatedBuildInputs = [
+    docopt
+    requests
+    beautifulsoup4
+  ];
+
+  checkInputs = [ black mypy flake8 ];
+
+  checkPhase = ''
+    echo -e "\x1b[32m## run black\x1b[0m"
+    LC_ALL=C black --check $(find -name \*.py)
+    echo -e "\x1b[32m## run flake8\x1b[0m"
+    flake8 hydracheck
+    echo -e "\x1b[32m## run mypy\x1b[0m"
+    mypy hydracheck
+  '';
+}
```

Comment:
To conform to [CONTRIBUTING.md#submitting-changes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes) please add a meta section with the follow attributes: `description`, `homepage` (can just be repo), `maintainer`, and `license`.

Reply:
Thanks for the hint! I've added the missing meta attributes